packages/patternfly-4/react-core/src/components/DataList/DataListToggle.tsx

```diff
@@ -1,17 +1,34 @@
-import React from 'react';
-import PropTypes from 'prop-types';
+import * as React from 'react';
 import { css } from '@patternfly/react-styles';
 import { AngleRightIcon } from '@patternfly/react-icons';
 import styles from '@patternfly/react-styles/css/components/DataList/data-list';
 import { Button, ButtonVariant } from '../Button';
+import { Omit } from '../../helpers/typeUtils';
 
-const DataListToggle = ({
-  className,
-  isExpanded,
-  'aria-controls': ariaControls,
-  'aria-label': ariaLabel,
-  'aria-labelledby': ariaLabelledBy,
-  rowid,
+export interface DataListToggleProps extends Omit<React.HTMLProps<HTMLDivElement>, 'aria-labelledby' | 'aria-label' | 'id'> {
```

Comment:
We don't need to exclude any of these attributes since the types are compatible (all `?: string`).